### ISSUE TITLE
Incorrect RPN for expressions with consecutive negations

### ISSUE THREAD

**Issue by victorjmarin** [dgorissen/pycel#105]
#### What actually happened

When parsing a valid Excel formula with two or more consecutive negations, the produced Reverse Polish Notation is incorrect. For example, the RPN returned for `ExcelFormula("=--1")` is `[OperatorNode<->, OperandNode<1>, OperatorNode<->]`.

#### What was expected to happen

The RPN expected for such an expression would be ` [OperandNode<1>, OperatorNode<->, OperatorNode<->]`.

#### Problem description

The current behavior is problematic because any other functionality relying on the RPN may fail. For example, the Abstract Syntax Tree construction relies on the RPN and thus fails for these kind of expressions.

#### Code Sample

```python
from pycel.excelformula import ExcelFormula
formula = ExcelFormula("=--1")
print(formula.rpn)
```

#### Environment

Pycel Version: 1.0b22
Python Version: 3.8.5
OS: MS Windows 10

**Comment by jpp-0** [comment]
Same issue, different project, but have provided an example workbook here: https://github.com/vallettea/koala/files/4934349/double_minus.xlsx

**Comment by victorjmarin** [comment]
Interestingly enough, I found that expressions like these are expected to fail. https://github.com/dgorissen/pycel/blob/master/tests/test_excelformula.py#L573

The reason why the expected RPN is not being generated resides in prefix unary operators being treated as left-associative. https://github.com/dgorissen/pycel/blob/master/src/pycel/excelformula.py#L137.
Prefix unary operators should be right-associative while postfix unary operators should remain left-associative.

I could try to contribute a fix if this is the expected behavior, but then again, the tests suggest against it.

**Comment by bradbase** [comment]
@jpp-0, I have included your spreadsheet in tests for @victorjmarin's contribution to xlcalculator.

Cheers to you both.

**Comment by bradbase** [comment]
Fix released in xlcalculator 0.2.12
